ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -99,6 +111,8 @@ class JITModBuilder {
 
     std::vector<CallInst *> inlined_calls;
     CallInst *last_call = nullptr;
+    size_t call_stack = 0;
```

Comment:
Would `StackDepth` or `CallDepth` be better? Either way, it needs to be C++ cased.

Reply:
(There are lots of places where the variable casing isn't correct)

Reply:
You mean LLVM cased. `CallDepth` works for me.